[EDU-3889] feat: add guide - new template (#762)
aziontech/docs: src/content/docs/pt-br/pages/guias/marketplace/templates/cdn-template.mdx

---

> title: Como implantar o template XXXXXX

Author: ```suggestion
title: Como implantar o template Dynamic and Static File Optimization
```

> description: Este template lhe ajuda criar uma aplicação com regras comuns de cache para melhorar a entrega do seu conteúdo.

Reviewer: ```suggestion
description: Este template lhe ajuda criar uma aplicação com políticas de cache padrão para melhorar a entrega do seu conteúdo.
```

Não sei se padrão é a melhor palavra, acho que "comuns" como estava antes também serve

meta_tags: templates, guias, Azion Marketplace

> namespace: docs_guides_XXXXX

Author: ```suggestion
namespace: docs_guides_dynamic_static_file_optimization
```

> permalink: /documentacao/produtos/guias/XXXXXXXX/

Author: ```suggestion
permalink: /documentacao/produtos/guias/dynamic-static-file-optimization-template/
```

---

import Badge from '~/components/Badge.astro';

<Badge variant="accent">
Preview
</Badge>

> O template **XXXXXXX** contém regras e configurações comuns de cache para melhorar a entrega e desempenho de conteúdo estático e dinâmico, bem como imagens, com base na extensão do arquivo. Além disso, ele também permite o processamento e otimização de imagens. Dessa forma, você pode entregar seu conteúdo usando a Azion como CDN.

Reviewer: ```suggestion
O template **XXXXXXX** contém políticas e configurações de cache padrão para melhorar a entrega e desempenho de conteúdo estático e dinâmico, bem como imagens, com base na extensão do arquivo. Além disso, ele também permite o processamento e otimização de imagens. Dessa forma, você pode entregar seu conteúdo usando a Azion como CDN.
```

> O template **XXXXXXX** contém políticas e configurações de cache padrão para melhorar a entrega e desempenho de conteúdo estático e dinâmico, bem como imagens, com base na extensão do arquivo. Além disso, ele também permite o processamento e otimização de imagens. Dessa forma, você pode entregar seu conteúdo usando a Azion como CDN.

Author: ```suggestion
O template **Dynamic and Static File Optimization** contém políticas e configurações de cache padrão para melhorar a entrega e desempenho de conteúdo estático e dinâmico, bem como imagens, com base na extensão do arquivo. Além disso, ele também permite o processamento e otimização de imagens. Dessa forma, você pode entregar seu conteúdo usando a Azion como CDN.
```

---

## Pré-requisitos

Para usar este template, você deve ativar:

<Badge>
[Application Acceleration](/pt-br/documentacao/produtos/build/edge-application/application-acceleration/)
</Badge>
<br/>
<Badge>
[Image Processor](/pt-br/documentacao/produtos/build/edge-application/image-processor/)
</Badge>

Você precisa habilitar esses módulos via Real-Time Manager (RTM). Para fazer isso:

1. Acesse o [Real-Time Manager (RTM)](/pt-br/documentacao/produtos/guias/como-acessar-o-rtm/) > **Account menu**.
2. Selecione a opção **Billing & Subscriptions**.
3. Selecione a aba **Subscriptions**.
4. Ative o switch para cada módulo.

:::caution[Atenção]
Se estes módulos estiverem ativados, a execução deste template pode gerar custos relacionados ao uso. Consulte a [página de preços](https://www.azion.com/pt-br/precos/) para mais informações.
:::

---

## Obtenha o template

Para obter o template, acesse o [Real-Time Manager (RTM)](/pt-br/documentacao/produtos/guias/como-acessar-o-rtm/) e selecione a opção **Start with a template** na página inicial.

> Isso abrirá uma nova página onde você pode selecionar o card do **xxxxx**.

Author: ```suggestion
Isso abrirá uma nova página onde você pode selecionar o card do **Dynamic and Static File Optimization**.
```

---

## Configure o template

> Uma vez que você está na página do template, clique na aba **Settings** para abrir o formulário de configuração e fornecer as informações para configurar sua aplicação.

Reviewer 2: ```suggestion
Na página do template, clique na aba **Settings** para abrir o formulário de configuração e fornecer as informações para configurar sua aplicação.
```

Campos identificados com um asterisco são obrigatórios.

- **Application Name** *: o nome da sua edge application na Azion.
    - Digite um nome único e fácil de lembrar. Se o nome já tiver sido usado, a plataforma retornará uma mensagem de erro.
- **Origin Address** *: o endereço da origem de onde a edge application deve buscar o conteúdo original.

>     - Exemplo: `www.yourwebsite.com`

Reviewer: ```suggestion
    - Exemplo: `www.yourwebsite.com`
```

Mesmo comentário anterior

> - **Custom Host Header**: em alguns casos, pode ser necessário adicionar informações do host no cabeçalho da solicitação para recuperar informações da origem. Esta variável é usada para esse propósito.

Reviewer: ```suggestion
- **Custom Host Header**: em alguns casos, pode ser necessário substituir o valor do cabeçalho `Host` na requisição para a origem. Defina um valor para o cabeçalho `Host` neste campo.
```

>     - Exemplo: `${host}`. Saiba mais sobre o host header e como customizá-lo no [documentação de Origins](/pt-br/documentacao/produtos/edge-application/origins/#host-header).

Reviewer: ```suggestion
    - Use a variável `${host}` para retirar o valor do cabeçalho `Host` da URI da requisição. Saiba mais sobre o host header e como customizá-lo no [documentação de Origins](/pt-br/documentacao/produtos/edge-application/origins/#host-header).
```
